web_sdk/web_test_utils/lib/skia_client.dart

```diff
@@ -132,7 +132,6 @@ class SkiaGoldClient {
       '--commit', commitHash,
       '--keys-file', keys.path,
       '--failure-file', failures.path,
-      '--passfail',
     ];
```

Comment:
On line 225, you may want to add a comment explaining the one remaining --passfail and why that is ok for tryjobs

Reply:
Done. Thanks for taking a look!